src/docs/operator-guides/how-to-edit-pages/index.md

```diff
@@ -0,0 +1,295 @@
+
+![Landing page sections in pages editor](/landing-page-sections.png 'Landing Page Sections')
+_Examples of sections you can create with Pages. You can decide to add new Sections, as well
+as editing or removing existing sections._
+
+### Define the section ID
+
+When creating a new section you need to define the section-id, or the
+name of the Section. It must be unique and should be descriptive. The
+clearer you make it, the easier it will be to identify this section
+later on and edit the Page in the future.
+
+**Important:** if you don’t select an identifier, you will not be able
+to save your landing page.
+
+### Select the Section template
+
+The Section template determines how Blocks are laid out in your section.
+There are four possible layouts:
+
+- Article: Narrow one column layout optimized for reading
+- Carousel: Displays content in blocks that can be scrolled through
+  horizontally.
+- Columns: Display content blocks in 1,2,3 or 4 column grid
+- Features: Text and media are displayed side by side in alternating
+  order
+
+You can see examples of each section template in our
+[Section templates article](https://www.sharetribe.com/docs/operator-guides/section-templates/)
+
+_Tip:_ The template can be modified later on, so don’t worry too much
+about it when testing,
+
+**Important:** if you don’t select a section template, you will not be
+able to save your landing page.
+
+### Add text content (Optional)
+
+You can add the title and description you want your users to see. The
+title will be displayed in bigger font at the top of the section. How
+big is determined by your title header level. H1 is bigger than H2. The
+title size also determines the HTML level header indexed by search
+engines. You can use the title to explain the content of the section or
+define a larger text that you want to highlight.
+
+The description refers to the subtitle of the section. It can be used as
+a short or long description for the section. These texts are optional.
+
+![Carousel image with title and description example](/title-and-description.png 'Title and description example')
+
+### Add call to action content (Optional)
+
+The call-to-action content is basically a button in your Section. The
+button can help you guide your users to specific actions that you want
+them to take. That is why it is named call to action. You can define the
+type of call to action, the text and the specific link.
+
+- Call to action type: No call to action, internal link or external
+  link. This determines the default behaviour and validation for the
+  button.
+  - **No call to action** means that there will be no general button for
+    this Section. You can always add buttons via call to action settings
+    withing blocks
+  - **Internal link** means that the link will open in the same tab. The
+    link should point to an internal folder within your marketplace,
+    with a partial URL.
+  - **External link** means that the link will open in a new tab. You
```

Comment:
"The link should point to a location outside your marketplace" should perhaps be in there?